lib/npm/test.js

```diff
@@ -65,6 +65,9 @@ function test(context, next) {
       nodeBin = which('node', {path: options.env.PATH || process.env.PATH});
     }
     var npmBin = which('npm',  {path: options.env.PATH || process.env.PATH});
+    if (windows) {
+      npmBin = path.join(path.dirname(npmBin), 'node_modules\\npm\\bin\\npm-cli.js');
```

Comment:
can you concatenate this file using path.join for the entire thing?
```
npmBin = path.join(path.dirname(npmBin), 'node_modules', 'npm', 'bin', 'npm-cli.js');
```
this will make the code more consistent with other places in the code base